internal/brokers/dbusbroker.go

```diff
@@ -116,38 +113,47 @@ func (b dbusBroker) IsAuthenticated(_ context.Context, sessionID, authentication
 
 // EndSession calls the corresponding method on the broker bus.
 func (b dbusBroker) EndSession(ctx context.Context, sessionID string) (err error) {
-	dbusMethod := DbusInterface + ".EndSession"
-
-	call := b.dbusObject.CallWithContext(ctx, dbusMethod, 0, sessionID)
-	if err = call.Err; err != nil {
-		return errmessages.NewErrorToDisplay(err)
+	if _, err := b.call(ctx, "EndSession", sessionID); err != nil {
+		return err
 	}
-
 	return nil
 }
 
 // CancelIsAuthenticated calls the corresponding method on the broker bus.
 func (b dbusBroker) CancelIsAuthenticated(ctx context.Context, sessionID string) {
-	dbusMethod := DbusInterface + ".CancelIsAuthenticated"
-
-	call := b.dbusObject.Call(dbusMethod, 0, sessionID)
-	if call.Err != nil {
-		log.Errorf(ctx, "could not cancel IsAuthenticated call for session %q: %v", sessionID, call.Err)
+	// We don’t want to cancel the context when the parent call is cancelled.
+	if _, err := b.call(context.Background(), "CancelIsAuthenticated", sessionID); err != nil {
+		log.Errorf(ctx, "could not cancel IsAuthenticated call for session %q: %v", sessionID, err)
```

Comment:
Unrelated to that PR, but I seems we don’t cover the error case for this one, mind adding a simple test in a separate PR to show that we ignore the error returned by dbus?